src/components/calcite-chip/calcite-chip.tsx

```diff
@@ -129,7 +130,7 @@ export class CalciteChip {
     );
 
     return (
-      <Host dir={dir}>
+      <Host class={{ [CSS_UTILITY.rtl]: rtl }}>
```

Comment:
@caripizza Can we add a container div and apply the class to that instead of the host?

That way the styles are internal to the component (shadow DOM) and can't be affected or removed by the end user.